singularity-eos/eos/eos_base.hpp

```diff
@@ -264,69 +296,13 @@ class EosBase {
   PORTABLE_INLINE_FUNCTION
   Real RhoPmin(const Real temp) const { return 0.0; }
 
-  // Specialzied vector version of PTofRE maybe more suited for EOSPAC
-  // }
-  // template<typename RealIndexer, typename LambdaIndexer>
-  // inline
-  // void PTofRE(RealIndexer &&rhos, RealIndexer &&sies,
-  //             RealIndexer &&presses, RealIndexer &&temps,
-  //             RealIndexer &&dpdrs, RealIndexer &&dpdes,
-  //             RealIndexer &&dtdrs, RealIndexer &&dtdes,
-  //             const int num, LambdaIndexer &&lambdas) const {
-  //   // Get the dervived class
-  //   auto eos = static_cast<CRTP const&>(*this);
-
-  //   // Lookup at density and energy
-  //   eos.PressureFromDensityInternalEnergy(rhos, sies, num, presses,
-  //                                               lambdas);
-  //   eos.TemperatureFromDensityInternalEnergy(rhos, sies, num, temps,
-  //                                                  lambdas);
-  //   // Peturbation factor
-  //   Real factor = 1. + 1.0e-06;
-
-  //   // Perturb densities and do lookups
-  //   portableFor(
-  //       'PerturbDensities', 0, num, PORTABLE_LAMBDA(const int i) {
-  //         rhos[i] *= factor;
-  //       }
-  //   );
-  //   eos.PressureFromDensityInternalEnergy(rhos, sies, num, dpdrs,
-  //                                               lambdas);
-  //   eos.TemperatureFromDensityInternalEnergy(rhos, sies, num, dtdrs,
-  //                                                  lambdas);
-
-  //   // Reset densities, perturb energies, and do lookups
-  //   portableFor(
-  //       'PerturbEnergiesResetDensities', 0, num,
-  //       PORTABLE_LAMBDA(const int i) {
-  //         sies[i] *= factor;
-  //         rhos[i] /= factor;
-  //       }
-  //   );
-  //   eos.PressureFromDensityInternalEnergy(rhos, sies, dpdes, num,
-  //                                               lambdas);
-  //   eos.TemperatureFromDensityInternalEnergy(rhos, sies, dtdes, num,
-  //                                                  lambdas);
-
-  //   // Reset the energies to their original values
-  //   portableFor(
-  //       'ResetEnergies', 0, num, PORTABLE_LAMBDA(const int i) {
-  //         sies[i] /= factor;
-  //       }
-  //   );
-
-  //   // Calculate the derivatives
-  //   portableFor(
-  //       'CalculateDerivatives', 0., num, PORTABLE_LAMBDA(const int i) {
-  //         dpdrs[i] = (dpdrs[i] - presses[i]) / (rhos[i] * (1. - factor));
-  //         dpdes[i] = (dpdes[i] - presses[i]) / (sies[i] * (1. - factor));
-  //         dtdrs[i] = (dtdrs[i] - temps[i]) / (rhos[i] * (1. - factor));
-  //         dtdes[i] = (dtdes[i] - temps[i]) / (sies[i] * (1. - factor));
-  //       }
-  //   );
-
-  //   return;
-  // }
+  // Default entropy behavior is to return an error
+  [[noreturn]] PORTABLE_FORCE_INLINE_FUNCTION
```

Comment:
If the compiler is still yelling at us for this decorator we could remove the `[[noreturn]]` from `portable_errors`... Not worth dealing with here but food for thought

Reply:
Hmmmm yeah maybe we do want to do this since it will only truly not return when the code aborts. When we throw an exception, the function should still return, right?

I'll change this.